sql_csv_utils.py

```diff
@@ -284,28 +284,6 @@ def create_update_statement(self, tab_name: str, col_list: list, val_list: list,
 
         return sql
 
```

Comment:
no longer useful sql_csv_tools method, no longer using a taxa_unmatch table to track TNRS flagged taxa.